packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -678,3 +674,8 @@ function blend(background: string, overlay: string, opacity: number, gamma: numb
     values: rgb as any,
   });
 }
+
+const removeOpacity = (color: string) => `rgb(from ${color} r g b / 1)`;
+function blendCssVars(background: string, overlay: string, opacity: number) {
+  return `color-mix(in srgb,${background}, ${removeOpacity(overlay)} calc(${opacity} * 100%))`;
```

Comment:
Wasn't sure if we could use `color-mix`. What are our supported browser versions?

Reply:
I wanted to validate this approach.
[`color-mix` requires iOS >= 16.2](https://caniuse.com/?search=color-mix), while we currently support >= 15.4. Same for Safari on MacOS. Other browsers seem to match what we have in our browserslist.

Do you have other ideas for solving this issue?

Reply:
No, this is the only thing that could work with CSS variables.